src/python/pants/ivy/ivy_subsystem.py

```diff
@@ -107,3 +121,13 @@ def repository_cache_dir(self):
       return self.get_options().repository_cache_dir
     else:
       return self.get_options().cache_dir
+
+
+class IvyUrlGenerator(BinaryToolUrlGenerator):
+
+  def __init__(self, template_urls):
+    super(IvyUrlGenerator, self).__init__()
```

Comment:
Ditto on Py3 style `super()`. I will add both these changes to the type checking PR I have up.